.nvmrc

```diff
@@ -0,0 +1 @@
+v10.19.0
```

Comment:
Doing so you can remove this setting from `.travis.yml` file. Reference: https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#specifying-nodejs-versions-using-nvmrc

I would just recommend you to comment the line and add the link as a source of documentation for us in the future instead of just deleting it and wait until magic happens hehehe